docs/development/intro.rst

```diff
@@ -112,23 +112,42 @@ Get set up
     Also, we shouldn't duplicate content from http://astropy.readthedocs.io/en/latest/#developer-documentation
     but link there instead.
 
-Commands to set up an environment for Gammapy development:
+The first steps are basically identical to https://cta-observatory.github.io/ctapipe/getting_started/index.html (until step 4, excluding 5)
+and http://astropy.readthedocs.io/en/latest/development/workflow/get_devel_version.html (up to *Create your own private workspace*).
+The following is a quick summary of commands to set up an environment for Gammapy development:
 
 .. code-block:: bash
 
+    # Fork the gammapy repository on GitHub, https://github.com/gammapy/gammapy
     cd code  # Go somewhere on your machine where you want to code
-    git clone https://github.com/gammapy/gammapy.git
+    git clone https://github.com/[your-github-username]/gammapy.git
     cd gammapy
     conda env create -f environment-dev.yml
     source activate gammapy-dev
+    # for conda versions >=4.4.0 you may have to execute
+    #'conda activate gammapy-dev' instead
+    git remote add gammapy git@github.com:gammapy/gammapy.git
+    git branch my-gammapy
```

Comment:
I find `my-gammapy` a bit confusing for a branch name. How about putting a `[branch-name]` placeholder like [here](https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf) or put something like `feature_branch_name` like [here](https://cta-observatory.github.io/ctapipe/getting_started/index.html#create-a-feature-branch).

Reply:
Added one line about branches and a reference to the *Make a working example* section, and used a `[branch-name]` placeholder.